### ISSUE TITLE
customized labels are not synced from backupSchedule CR to backup CR

### ISSUE THREAD

**Issue by DanielZhangQD** [pingcap/tidb-operator#2345]
## Feature Request

**Is your feature request related to a problem? Please describe:**

**Describe the feature you'd like:**

Currently, TiDB Operator generates the labels automatically for the backupSchedule CR  and syncs them to the backup CR, however, customized labels are not synced to the backup CR.

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

**Comment by weekface** [comment]
Annotations has been synced from BackupSchedule to Backup. Backup has these labels:

```
      app.kubernetes.io/instance: test1
      tidb.pingcap.com/backup-schedule: test1
      app.kubernetes.io/managed-by: backup-schedule-operator
      app.kubernetes.io/name: backup-schedule
```

Users can use these labels to filter them.

**Comment by weekface** [comment]
@DanielZhangQD @gejibin Please take a look.

**Comment by DanielZhangQD** [comment]
> Annotations has been synced from BackupSchedule to Backup. Backup has these labels:
> 
> ```
>       app.kubernetes.io/instance: test1
>       tidb.pingcap.com/backup-schedule: test1
>       app.kubernetes.io/managed-by: backup-schedule-operator
>       app.kubernetes.io/name: backup-schedule
> ```
> 
> Users can use these labels to filter them.

Yes, I agree.

**Comment by tirsen** [comment]
Will this issue be fixed by https://github.com/pingcap/tidb-operator/pull/3892 ?

**Comment by DanielZhangQD** [comment]
> Will this issue be fixed by #3892 ?

@tirsen Yes, we will cover this in #3892.
